services/frontend/src/utils/models/Profile.test.js

```diff
@@ -1,5 +1,20 @@
 import {Profile, ProfileSkill, Skill} from "utils/models";
 
+/* Mock the date so "createdAt" and "updatedAt" properties will not be invalid if created at different times */
```

Comment:
Maybe my surface is just runs slow or something but sometimes it will have a few of the skills be created 0.001 seconds apart from each other. 

And yes, I can just pass through date objects if you would rather that.
```js
const skillA = new Skill({id: "x", name: "test"});
or
const skillA = new Skill({id: "x", name: "test", ...constantDates});
```
However, in the profile tests the date object is never used functionally and has no effect on validity. I can agree it's maybe excessive to mock the entire global date object but is this not one of the better cases for us to mock it in some respect?

Reply:
Man, that timing thing is _really_ weird. Gotta pump them gigahertz up... If that's the case, eh, just leave it.